Troubleshooting.md

```diff
@@ -0,0 +1,77 @@
+# Troubleshooting
+
+This document will help troubleshoot the common debugging errors.
+
+## Java Language Support extension is not started or broken.
+Java Debugger is leveraging `"Language Support for Java(TM) by Red Hat"` extension for source mapping and project support. If Java Language Support extension is not started or broken, Java Debugger may not work as expected. Here is a simple way to check the Language Server is started. Open a Java or pom.xml file, and then check the status icon at the right corner of VS Code status bar is thumbs up or not.
+
+ ![ status indicator ](https://raw.githubusercontent.com/redhat-developer/vscode-java/master/images/statusMarker.png).

```

Comment:
add jdk trouble shoot like: check whether JAVA_HOME is in system variables, make sure the `which/where java`  is pointing to the right jdk.

Reply:
resolved.